src/libraries/System.Linq.Expressions/tests/Array/ArrayArrayIndexTests.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace System.Linq.Expressions.Tests
 {
+    [ActiveIssue("", TestPlatforms.tvOS)]
```

Comment:
this needs to have a GitHub issue (there are a bunch more of these).

Reply:
Thanks for the catch! :D